Appraisals

```diff
@@ -1,11 +1,11 @@
-case(RUBY_VERSION)
+case RUBY_VERSION
 
-  when '2.3.3', '2.5.3' then
+when '2.3.3', '2.5.3'
```

Comment:
What about `2.6.3`?

Reply:
I added it, but it was on a later commit.